dpe/src/state.rs

```diff
@@ -9,6 +9,7 @@ use crate::{
 };
 use bitflags::bitflags;
 use caliptra_cfi_lib_git::cfi_launder;
+use core::mem::align_of;
```

Comment:
This was failing to build for some reason in `caliptra-sw`. Not sure how it doesn't get flagged in this repo

Reply:
It is failing this line which should be built with or without CFI:

```Rust
const _: () = assert!(align_of::<State>() == 4);
```

Reply:
Sorry - didn't delete my earlier comment in time, thanks for clarifying! (I commented too fast).